src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -12,7 +12,7 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{5471BFE8-8071-466F-838E-5ADAA779E742}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <UseECMAKey Condition="'$(UseECMAKey)'==''">true</UseECMAKey>
```

Comment:
OK if that is the case then we should probably also remove the override for this in dir.props. However I'm fine with that being a follow-up item.